pkg/server/conn.go

```diff
@@ -392,7 +395,8 @@ func closeConn(cc *clientConn, connections int) error {
 
 func (cc *clientConn) closeWithoutLock() error {
 	delete(cc.server.clients, cc.connectionID)
-	return closeConn(cc, len(cc.server.clients))
+	delete(cc.server.resourceGroupMap, cc.getCtx().GetSessionVars().ResourceGroupName)
```

Comment:
shoud it be `cc.server.resourceGroupMap[cc.getCtx().GetSessionVars().ResourceGroupName]--` here? you may delete this key if the value reaches 0

Reply:
no, the resource group map only record the <resource group name , connection count>

Reply:
good catch, I will dec this count not delete.